### ISSUE TITLE
Compliance/Compliance summary view - pending penalties

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-compliance#99]
#### Description:

As an operator, I can see automatic overdue penalties that are accruing because the overdue obligation has not yet been paid. When my obligation I paid, I no longer see this section.

[Design in Figma](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=20107-1258&t=D4UaBo6pmGx65d5s-1)

- [ ] Header: Penalty (as of today)

Automatic overdue penalty details:
- [ ] Notice that the penalty will increase until the compliance obligation is paid.
- [ ] Penalty status
- [ ] Penalty type
- [ ] Penalty rate (daily): 0.38% compounding
- [ ] Accumulated penalty 
- [ ] Accumulated compounding
- [ ] Penalty (as of today)
- [ ] FAA interest (as of today)
- [ ] Total amount (as of today)

Data source:
See penalty tickets for details.

#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking
Blocked by #18 

#### Notes:

-

**Comment by nanyangpro** [comment]
Hi @NicoleGovvy - Thanks for pinging me and your suggested fields for the "Additional unknown obligation penalty details"!
- I noticed that you added "Penalty issued date" and "Penalty due date" fields in the "Penalty (as of today)" section, which is not aligned with the design. Shall we remove those 2 fields from the ticket description? Users are not supposed to see those fields on the Compliance Summary page, the dates will only be settled after the "Penalty Status" changes from "Accruing" to "Due", i.e. after compliance obligation is fully met.

**Comment by NicoleGovvy** [comment]
@nanyangpro ah thanks for the feedback. I had imagined those dates would only be displayed once the obligation has been met, so the penalty is a final, known amount and not accruing. That is not clear from the field list I see now.

**Comment by nanyangpro** [comment]
- Please refer to the [Penalty Summary page](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=23398-27&t=4totV6YYEmE2ndSh-4) for **when the Automatic Overdue page is actually settled**
- The "Penalty (as of Today)" section on the [Compliance Summary page](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=20107-1258&t=4totV6YYEmE2ndSh-4) serves slightly different goal from the above
- I added a 2nd point in my first comment above for more context of wht those dates are not in the design :)

**Comment by NicoleGovvy** [comment]
I see that the penalty grid shows 30 days for the penalty to be paid, so I am a bit confused. This would be good to confirm with the business area. 

My assumption was that the penalty today section on the compliance summary page, once the obligation is met/penalty no longer in accrual state, would then convert to show details about the final penalty status, and could also be viewed on the penalty summary page as you provided the link to above. 

So I am not clear on what the penalty (as of today) section will show then, once the penalty amount is final!

**Comment by NicoleGovvy** [comment]
@nanyangpro I've edited this ticket to reflect the accuring penalty information.

**Comment by vesselak** [comment]
This will be completed as part of #74 UI part of the ticket
